apps/backend/src/controllers/property.controller.ts

```
@@ -741,3 +791,41 @@ export async function updatePropertyStatusController(
+      res
+        .status(500)
+        .json(
+          formatErrorResponse(
+            result.error ?? 'Unknown error occurred',
+            result.details
+          )
+        );
+      return;
+    }
+
+    const response = result.data!.map((property: any) => ({
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)